Comments.tmPreferences

```diff
@@ -1,48 +1,37 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>name</key>
-	<string>Comments</string>
-	<key>scope</key>
-	<string>source.julia</string>
-	<key>settings</key>
-	<dict>
-		<key>shellVariables</key>
-		<array>
-			<dict>
-				<key>name</key>
-				<string>TM_COMMENT_START</string>
-				<key>value</key>
-				<string># </string>
```

Comment:
The trailing space got lost. Please, merge [this PR](https://github.com/JuliaEditorSupport/Julia-sublime/pull/102), which re-adds it.